sched/group/group_signal.c

```diff
@@ -136,7 +139,7 @@ static int group_signal_handler(pid_t pid, FAR void *arg)
 
       /* Is this signal unblocked on this thread? */
 
-      if (!sigismember(&tcb->sigprocmask, info->siginfo->si_signo) &&
+      if (!nxsig_ismember(&tcb->sigprocmask, info->siginfo->si_signo) &&
```

Comment:
```suggestion
      ret = nxsig_ismember(&tcb->sigprocmask, info->siginfo->si_signo)
      if (ret == 0 &&
```

Reply:
Hmmm... In this case the signal number should be okay. Probably a DEBUGASSERT(GOOD_SIGNO( info->siginfo->si_signo)) would give us sufficient protection.